<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calcHistoryDialog</class>
 <widget class="QDialog" name="calcHistoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>370</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Calculations history</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/dalculator.ico</normaloff>:/icons/dalculator.ico</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0">
   <item>
    <widget class="QListWidget" name="listHistory">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonCurrExpr">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Current expression</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutDelete">
     <item>
      <widget class="QPushButton" name="buttonRemove">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClearHistory">
       <property name="text">
        <string>Clear history</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutUseClose">
     <item>
      <widget class="QPushButton" name="buttonUse">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Use</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
